src/vs/vscode.d.ts

```diff
@@ -1896,6 +1896,13 @@ declare module 'vscode' {
 		resolveCodeLens?(codeLens: CodeLens, token: CancellationToken): ProviderResult<CodeLens>;
 	}
 
+	export class DefinitionAndSpan {
```

Comment:
We may want to make this use a single definition instead:

```ts
 class DefinitionAndSpan {
 		span: Location;
 		definition: Location;
}
```

My original thinking was to bundle the concept of a definition into a class, i.e. this text range corresponds to these X definitions. However I'm not sure this makes sense since we later flatten out the definition list in the UI anyways